cmd/emit.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"github.com/boson-project/func/cloudevents"
+	"github.com/google/uuid"
+	"github.com/ory/viper"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	e := cloudevents.NewEmitter()
+	root.AddCommand(emitCmd)
+	emitCmd.Flags().BoolP("local", "l", false, "Send the CloudEvent to the function running on the local host")
+	emitCmd.Flags().StringP("source", "s", e.Source, "CloudEvent source")
```

Comment:
Would be nice to allow sending to other Sinks (not just a function) as well. I can see a very common ask to send to the Broker/Channel as well.

Reply:
Currently I'm proposing a `--local` boolean flag. To support arbitrary endpoints, this should probably be a little different semantically.

```
func emit [--sink=endpoint]
```
When `--sink` is omitted, the event is by default sent to the deployed function. This means that to send an event to a function running locally via `func run`, the command would be

```
funk emit --sink=http://localhost:8080
```
A possibility is to have a special value for local, e.g. 

```
func emit --sink=local
```

WDYT?

Reply:
The reason why I want to have the possibility to send to arbitrary endpoints is that when I am developing a function, I might want to test the events flow in my system, eg. that all triggers are correctly set etc. So in that case I don't want to send the CE directly to the function, but mimic what would happen in the real deployment, for example send it to the broker.

And why not make the UX consistent? If this is not implemented, the user would have to switch to different tool (kn-event-plugin) which could have a slightly different syntax and whatnot.

Reply:
@zroubalik so concretely, are you suggesting something like a `--endpoint` flag that, when provided, overrides any `--path` or `$FUNC_PATH` values and ignores `cwd()`?

Reply:
Exactly and I like what you proposed above `--sink=local` or `--endpoint=local`, this way we can omit the `-l` flag.